### ISSUE TITLE
Are you looking for maintainers?  

### ISSUE THREAD

**Issue by GonzaRodriguez** [bhgames/json-logic-ruby#33]
I noticed that the repo is not being actively maintained for quite a long time. I would be pleased to dedicate same time to help in the maintenance of this gem if you agreed. Please let me know what do you think @bhgames.

**Comment by iamzayn19** [comment]
Same as my note on #44 — not an official maintainer here, but I've fixed the outstanding bug reports in a fork (295/295 official test suite passing): https://github.com/iamzayn19/json-logic-ruby. Open to helping maintain if that'd be useful.

**Comment by iamzayn19** [comment]
Following up here too — published the fixes as a gem: [`jlogic_ruby`](https://rubygems.org/gems/jlogic_ruby). Not trying to step on anyone's toes, just didn't want the open bugs to sit unfixed. Credit to the original authors for the library itself — I'll keep maintaining the fork for anyone who needs it.

**Comment by iamzayn19** [comment]
Update — @bhgames just merged a batch of the open fixes (#26, #30, #34, #37, #40, #48, #51), so definitely not abandoned. 🎉

**Comment by bhgames** [comment]
@iamzayn19 you want to be added as a maintainer here? I haven't touched ruby in a decade really. Happy to add you.

**Comment by iamzayn19** [comment]
@bhgames Yes, I'd love to! Happy to keep an eye on issues/PRs and keep the fixes flowing. Thanks for the trust 🙏
